documentation/content/en/books/porters-handbook/upgrading/_index.adoc

```diff
@@ -149,6 +149,8 @@ If adding, moving, or removing a file, use `git` to track these changes:
 
 Make sure to check the port using the checklist in crossref:quick-porting[porting-testing,Testing the Port] and crossref:quick-porting[porting-portlint,Checking the Port with `portlint`].
 
+Also, if the source files were changed, update the checksum reference in distinfo with `make makesum`.
```

Comment:
1- Thanks for this.

2- distinfo (checksum and size) depends on the tarball itself, not individual files, so I'd remove "if ... were changed". Besides, it can't hurt to do it every time. Worst that can happen is recreating distinfo with the same contents it had.